spring-cloud-config-client/.gitignore

```diff
@@ -0,0 +1 @@
+/.apt_generated/
```

Comment:
It was generated somewhere. I did not add it. I googled and it says that that is where code generated by Annotations is placed. And I do believe that there is code generated by Lombok in there. So it is probably a good idea, but I can remove it if that is preferable. There are a couple of .apt_generated folders generated by the build on my machine.